stdlib/public/core/RangeReplaceableCollection.swift.gyb

```diff
@@ -1171,6 +1142,52 @@ public func +<
   return lhs
 }
 
+/// Appends the elements of a sequence to a range-replaceable collection.
+///
+/// Use this operator to append the elements of a sequence to the end of
+/// range-replaceable collection with same `Element` type. This example appends
+/// the elements of a `Range<Int>` instance to an array of integers.
+///
+///     var numbers = [1, 2, 3, 4, 5]
+///     numbers += 10...15
+///     print(numbers)
+///     // Prints "[1, 2, 3, 4, 5, 10, 11, 12, 13, 14, 15]"
+///
+/// - Parameters:
+///   - lhs: The array to append to.
+///   - rhs: A collection or finite sequence.
+///
+/// - Complexity: O(*n*), where *n* is the length of the resulting array.
+public func += <
+  R : RangeReplaceableCollection, S : Sequence
+>(lhs: inout R, rhs: S) 
+ where R.Iterator.Element == S.Iterator.Element {
+  lhs.append(contentsOf: rhs)
+}
+
+/// Appends the elements of a collection to a range-replaceable collection.
+///
+/// Use this operator to append the elements of a collection to the end of
+/// range-replaceable collection with same `Element` type. This example appends
+/// the elements of a `Range<Int>` instance to an array of integers.
+///
+///     var numbers = [1, 2, 3, 4, 5]
+///     numbers += 10...15
+///     print(numbers)
+///     // Prints "[1, 2, 3, 4, 5, 10, 11, 12, 13, 14, 15]"
+///
+/// - Parameters:
+///   - lhs: The array to append to.
+///   - rhs: A collection.
+///
+/// - Complexity: O(*n*), where *n* is the length of the resulting array.
+public func += <
```

Comment:
Since these two `+=` operators are new APIs, this PR needs tests.  (Use minimal collections from StdlibCollectionUnittest.)